web/static/js/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux'
+
+import user from './user'
+
+
```

Comment:
lot of extra whitespace in these files, especially after imports. only one line is necessary